pkg/cloud/azure/assets/validating-admission-service-annotation-policy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: azure-load-balancer-tcp-idle-timeout-annotation-validation-policy
+spec:
+  matchConstraints:
+    resourceRules:
+      - apiGroups: [""]
+        apiVersions: ["v1"]
+        operations: ["CREATE", "UPDATE"]
+        resources: ["services"]
+  validations:
+    - expression: |
```

Comment:
Do we need to ratchet this validation as you have done on the GCP version?

Reply:
yes I think so , will update this and validate.